extensions/chocolatey-core.extension/extensions/Install-ChocolateyInstallPackage.ps1

```diff
@@ -0,0 +1,104 @@
+<#
+.SYNOPSIS
+Simple wrapper to support easier selection of 32bit and 64bit install file.
+**NOTE:** Administrative Access Required.
+.DESCRIPTION
+This will select the appropriate installer depending on the system environment
+and the user passed arguments.
+.PARAMETER PackageName
+The name of the package - while this is an arbitrary value, it's
+recommended that it matches the package id.
+.PARAMETER FileType
+This is the extension of the file. This can be 'exe', 'msi', or 'msu'.
+.PARAMETER SilentArgs
+OPTIONAL - These are the parameters to pass to the native installer,
+including any arguments to make the installer silent/unattended.
+.PARAMETER File
+Full file path to the native 32bit installer to run.
+OPTIONAL when File64 have been set.
+.PARAMETER File64
+Full file path to the native 64bit installer to run.
+OPTIONAL when File have been set.
+.PARAMETER ValidExitCodes
+Array of exit codes indicating success. Defaults to `@(0)`.
+.PARAMETER UseOnlyPackageSilentArguments
+Do not allow choco to privde/merge additional silent arguments and
+only use the ones available with the package. Available in 0.9.10+.
+.PARAMETER IgnoredArguments
+Allows splatting with argumnets that do not apply. Do not use directly.
+.EXAMPLE
+$packageName= 'bob'
+$toolsDir   = "$(Split-Path -Parent $MyInvocation.MyCommand.Definition)"
+$fileLocation = Join-Path $toolsDir '32BIT_INSTALLER_EMBEDDED_IN_PACKAGE'
+$fileLocation64 = Join-Path $toolsDir '64BIT_INSTALLER_EMBEDDED_IN_PACKAGE'
+
+$packageArgs = @{
+  packageName   = $packageName
+  fileType      = 'msi'
+  file          = $fileLocation
+  file64        = $fileLocation64
+  silentArgs    = "/qn /norestart"
+  validExitCodes= @(0, 3010, 1641)
+  softwareName  =
+}
+
+Install-ChocolateyInstallPackageEx @packageArgs
+#>
+$have64BitFileParam = Get-Item function:\Install-ChocolateyInstallPackage | ? { $_.Parameters.ContainsKey('file64') }
+if ($have64BitFileParam) {
+  return
+}
+
+Write-Debug "Loading Install-ChocolateyInstallPackage override"
```

Comment:
And these debugging messages are necessary for determining issues - it may not be a bad idea to also have a debug message in the function itself.